src/testimonials/index.js

```diff
@@ -2,6 +2,7 @@ export const enigmaTestimonial = {
   name: 'Harley Sugarman',
   title: 'Founder, CEO at Enigma',
   imgSrc: 'harley',
+  svgAlt: 'Enigma',
```

Comment:
Perhaps we should call this `companyName` or something? Then it's a little more generic for use elsewhere.